timber-lint/src/main/java/timber/lint/WrongTimberUsageDetector.java

```diff
@@ -548,7 +548,10 @@ private boolean isLoggingExceptionMessage(JavaContext context, UExpression arg)
     UQualifiedReferenceExpression argExpression = (UQualifiedReferenceExpression) arg;
     UExpression selector = argExpression.getSelector();
 
-    // what other UExpressions could be a selector?
+    if (!(selector instanceof UCallExpression)) {
```

Comment:
I don't know much about the lint api. Is this the right fix? Is there selectors other than `UCallExpression` we care about.

Reply:
relates to #311. looks like we also care about `KotlinUSimpleReferenceExpression`

Reply:
updated above to also inspect `KotlinUQualifiedReferenceExpression`

Reply:
re: other selectors, I'm not sure.  Perhaps leave the comment there?